pinc/charsuite-polytonic-greek.inc

```diff
@@ -4,87 +4,87 @@ include_once($relPath."CharSuites.inc");
     'U+0375',
-    'U+003B',
-    'U+00B7',
-    'U+0391-U+03A1',
-    'U+03A3-U+03A9',
-    'U+03AA-U+03AB',
-    'U+03B1-U+03C9',
-    'U+03CA',
-    'U+03CB',
-    'U+03DB',
-    'U+03DC',
-    'U+03DD',
-    'U+03F2',
-    'U+03F9',
+    'U+003b',
+    'U+00b7',
+    'U+0391-U+03a1',
+    'U+03a3-U+03a9',
+    'U+03aa-U+03ab',
+    'U+03b1-U+03c9',
+    'U+03ca',
+    'U+03cb',
+    'U+03db',
+    'U+03dc',
+    'U+03dd',
+    'U+03f2',
+    'U+03f9',
     # tonos accented characters
     'U+0386',
     'U+0388',
     'U+0389',
-    'U+038A',
-    'U+038C',
-    'U+038E',
-    'U+038F',
+    'U+038a',
+    'U+038c',
+    'U+038e',
+    'U+038f',
     'U+0390',
-    'U+03AC',
-    'U+03AD',
-    'U+03AE',
-    'U+03AF',
-    'U+03B0',
-    'U+03CC',
-    'U+03CD',
-    'U+03CE',
+    'U+03ac',
+    'U+03ad',
+    'U+03ae',
+    'U+03af',
+    'U+03b0',
+    'U+03cc',
+    'U+03cd',
+    'U+03ce',
     # polytonic characters
-    'U+1F00-U+1F15',
-    'U+1F18-U+1F1D',
-    'U+1F20-U+1F45',
-    'U+1F48-U+1F4D',
-    'U+1F50-U+1F57',
-    'U+1F59',
-    'U+1F5B',
-    'U+1F5D',
-    'U+1F5F',
-    'U+1F60-U+1F70',
-    # skip oxia U+1F71
-    'U+1F72',
-    # skip oxia U+1F73
-    'U+1F74',
-    # skip oxia U+1F75
-    'U+1F76',
-    # skip oxia U+1F77
-    'U+1F78',
-    # skip oxia U+1F79
-    'U+1F7A',
-    # skip oxia U+1F7B
-    'U+1F7C',
-    # skip oxia U+1F7D
-    'U+1F80-U+1FB4',
-    'U+1FB6-U+1FBA',
-    # skip oxia U+1FBB
-    'U+1FBC',
-    'U+1FC2-U+1FC4',
-    'U+1FC6-U+1FC8',
-    # skip oxia U+1FC9
-    'U+1FCA',
-    # skip oxia U+1FCB
-    'U+1FCC',
-    'U+1FD0-U+1FD2',
-    # skip oxia U+1FD3
-    'U+1FD6-U+1FDA',
-    # skip oxia U+1FDB
-    'U+1FE0-U+1FE2',
-    # skip oxia U+1FE3
-    'U+1FE4-U+1FEA',
-    # skip oxia U+1FEB
-    'U+1FEC',
-    'U+1FF2-U+1FF4',
-    'U+1FF6-U+1FF8',
-    # skip oxia U+1FF9
-    'U+1FFA',
-    # skip oxia U+1FFB
-    'U+1FFC',
+    'U+1f00-U+1f15',
+    'U+1f18-U+1f1d',
+    'U+1f20-U+1f45',
+    'U+1f48-U+1f4d',
+    'U+1f50-U+1f57',
+    'U+1f59',
+    'U+1f5b',
+    'U+1f5d',
+    'U+1f5f',
+    'U+1f60-U+1f70',
+    # skip oxia U+1f71
+    'U+1f72',
+    # skip oxia U+1f73
+    'U+1f74',
+    # skip oxia U+1f75
+    'U+1f76',
+    # skip oxia U+1f77
+    'U+1f78',
+    # skip oxia U+1f79
+    'U+1f7a',
+    # skip oxia U+1f7b
+    'U+1f7c',
+    # skip oxia U+1f7d
+    'U+1f80-U+1fb4',
+    'U+1fb6-U+1fba',
+    # skip oxia U+1fbb
+    'U+1fbc',
+    'U+1fc2-U+1fc4',
+    'U+1fc6-U+1fc8',
+    # skip oxia U+1fc9
+    'U+1fca',
+    # skip oxia U+1fcb
+    'U+1fcc',
+    'U+1fd0-U+1fd2',
+    # skip oxia U+1fd3
+    'U+1fd6-U+1fda',
+    # skip oxia U+1fdb
+    'U+1fe0-U+1fe2',
+    # skip oxia U+1fe3
+    'U+1fe4-U+1fea',
+    # skip oxia U+1feb
+    'U+1fec',
+    'U+1ff2-U+1ff4',
+    'U+1ff6-U+1ff8',
+    # skip oxia U+1ff9
+    'U+1ffa',
+    # skip oxia U+1ffb
+    'U+1ffc',
 ];
 $charsuite->reference_urls = [
     "https://www.pgdp.net/wiki/DP_Code_-_Unicode/Greek",
```

Comment:
not sure the usual approach here, but should we use site vars to make this not pgdp.net specific?

Reply:
Good question. In this case the above-reference page goes into more detail about the complications when creating a Polytonic Greek character suite so it will be useful for other consumers too.

Reply:
Would it be better to have it as a markdown file within the code base? (Or protect the wiki page and document why it should stay where it is.)

Reply:
It's fine where it is. I'm not too worried about someone coming in and changing the wiki page, although if we wanted to protect it that wouldn't be a bad idea.